app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  validates :name, presence: true,
+length: {maximum: Rails.application.config.maximum_name_length}
```

Comment:
```suggestion
  validates :name, presence: true,
    length: {maximum: Rails.application.config.maximum_name_length}
```
sai indent nè cu, như chị suggest mới đúng, em có run rubocop ko thế :D


tự fix mấy line bên duới nhá

Reply:
cơ mà thay vì khai bào trong file application.rb, em tìm hiểu về gem config đi em, nó sẽ gen file settings cho em đó, và các giá trị hằng này mình khai báo tập trung trong file config luôn gái 
https://github.com/rubyconfig/config?tab=readme-ov-file#installing